src/components/LandmarkDrawer.tsx

```diff
@@ -118,7 +147,12 @@ function LandmarkAssessmentHistory({ landmark } : {
         <Th>详情</Th>
       </Tr>
       <Tbody>
-        {assessments?.map((assessment, index) => (
+        {assessments?.sort((a1, a2) => {
+          const date1 = new Date(a1.createdAt ?? "");
+          const date2 = new Date(a2.createdAt ?? "");
+          return date2.getTime() - date1.getTime();  
+        })
```

Comment:
extract the sorting function in src/components/ChatRoom.tsx:36 into a utility function in strings.ts and reuse this function.